tests/test_a2c_conv.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import tensorflow as tf
+import numpy as np
+from stable_baselines.a2c.utils import conv
+from stable_baselines.common.cmd_util import make_atari_env
+from stable_baselines.common.vec_env import VecFrameStack
+from stable_baselines.common.input import observation_input
+
+ENV_ID = 'BreakoutNoFrameskip-v4'
+SEED = 3
+NUM_TIMESTEPS = 500
+NUM_CPU = 2
+
+
+@pytest.mark.parametrize("input1", [4])
+@pytest.mark.parametrize("input2", [(3, 5)])
+def test_conv_kernel(input1, input2):
```

Comment:
Why do you use parametrize here?

Reply:
Is there any drawback to use parametrize? If there is a better way, I am happy to change it.

Reply:
well, parametrize is useful when you want to test several values..., here you are using only one, so it does not really make sense.

Reply:
also, you should use variable name that are meaningful, (eg filter_size_1 instead of input_1)